docs/recipes/Adding-Styles.md

```diff
@@ -2,6 +2,171 @@
 
 # Adding Styles
 
-Recipe is forthcoming.
+## 📖 Table of Contents
 
-[☝️ Return To Top](#adding-styles)
\ No newline at end of file
+* [CSS Modules](#css-modules)
+  * [Example Class Names](#example-class-names)
+  * [Importing and Using Classes](#importing-and-using-classes)
+* [Adding a CSS File to Root Module](#adding-a-css-file-to-root-module)
+* [Adding a CSS File to Child Module](#adding-a-css-file-to-child-module)
+* [Using Reakit with One App](#using-reakit-with-one-app)
+  * [Add Reakit to Root Module](#add-reakit-to-root-module)
+  * [Using Reakit in a Child Module](#using-reakit-in-a-child-module)
+  * [Share Reakit Across Your Modules](#share-reakit-across-your-modules)
+    * [Add Reakit as a `providedExternals` in your Root Module](#add-reakit-as-a-providedexternals-in-your-root-module)
+    * [Add Reakit as a `requiredExternals` in your Child Module](#add-reakit-as-a-requiredexternals-in-your-child-module)
+
+## CSS Modules
+
+Since CSS is global, One App uses the [CSS Modules](https://github.com/css-modules/css-modules) pattern to avoid class name collisions. The main differences will be the strict use of camelCased class names and the way you import the class names.
```

Comment:
Could we break this up to 100 chars per line, and anywhere else in the doc please